android/src/main/java/com/tailscale/ipn/App.java

```diff
@@ -94,6 +103,27 @@ String getHostname() {
 		return manu + " " + model;
 	}
 
+	// get user defined nickname from Settings
+	// returns null if not available
+	private String getUserConfiguredDeviceName() {
+		String nameFromSystemBluetooth = Settings.System.getString(getContentResolver(), "bluetooth_name");
+		String nameFromSecureBluetooth = Settings.Secure.getString(getContentResolver(), "bluetooth_name");
+		String nameFromSystemDevice = Settings.Secure.getString(getContentResolver(), "device_name");
+
+		Log.d("com.tailscale.ipn.App", "Device name from System Bluetooth: " + nameFromSystemBluetooth);
+		Log.d("com.tailscale.ipn.App", "Device name from Secure Bluetooth: " + nameFromSecureBluetooth);
+		Log.d("com.tailscale.ipn.App", "Device name from System Device: " + nameFromSystemDevice);
```

Comment:
Let's drop the logging until they're needed. Also, only the Go side of the app is logging right now, avoiding the need for updating the log tag in multiple places.

Reply:
Sounds good. Removed!